plugins/org.eclipse.elk.alg.rectpacking/META-INF/MANIFEST.MF

```diff
@@ -11,4 +11,5 @@ Require-Bundle: org.eclipse.elk.graph,
  com.google.guava,
  org.eclipse.xtext.xbase.lib;bundle-version="2.10.0"
 Export-Package: org.eclipse.elk.alg.rectpacking,
+ org.eclipse.elk.alg.rectpacking.graphvisitors,
```

Comment:
I'd introduce a simple `graph` package to follow conventions established by Layered and other algorithms.

Reply:
I will add it to the root package as suggested by @uruuru